sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -5570,6 +5570,21 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSparkSession {
       Seq(Row(Seq("b", "a", "c", null)))
     )
   }
+
+  test("function current_catalog, current_database, current_schema") {
+    val df = Seq((1, 2), (3, 1)).toDF("a", "b")
```

Comment:
nit: can we just use `spark.emptyDataFrame`?

Reply:
I don't think so, in that case `spark.emptyDataFrame.select*` is also empty, and then the results comparison is always true (since both df are empty)

Reply:
Got it , thanks